api/src/main/java/com/lightbend/lagom/javadsl/api/Descriptor.java

```diff
@@ -375,6 +389,39 @@ public String toString() {
         }
     }
 
+    /**
+     * Describes a topic call.
+     */
+    public static interface TopicCall<Message> {
```

Comment:
The concrete class is internal (see InternalTopicCall). I've done things this way because the TopicCall is part of the public api (even though users should never use it directly), hence it contains only getters. While the InternalTopicCall also has setters.